code/src/layouts/modals/auth/components/sign-in-form/SignInForm.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { screen, fireEvent, waitFor, act } from "@testing-library/react";
+import SignInForm from "@/layouts/modals/auth/components/sign-in-form/SignInForm";
+import useSignIn from "@/hooks/use-sign-in/useSignIn";
+import { renderWithProviders } from "@/utils/test-utils";
+
+jest.mock("@/hooks/use-sign-in/useSignIn", () => ({
+  __esModule: true,
+  default: jest.fn()
+}));
+
+const mockSignIn = jest.fn();
+(useSignIn as jest.Mock).mockReturnValue([mockSignIn, { isLoading: false }]);
+
+const mockFormValues = {
+  email: "test@example.com",
+  password: "Helloworld123!"
+};
+
+describe("SignInForm", () => {
+  beforeEach(() => {
+    renderWithProviders(<SignInForm />);
+  });
+
+  test("renders the form correctly", () => {
+    const passwordInput = screen.getByLabelText(/signIn.password.field/i);
+    const emailInput = screen.getByLabelText(/signIn.email.field/i);
+    const submitButton = screen.getByRole("button", { name: /signIn.button/i });
+
+    expect(passwordInput).toBeInTheDocument();
+    expect(emailInput).toBeInTheDocument();
+    expect(submitButton).toBeInTheDocument();
+  });
+
+  test("handles input changes and form submission", async () => {
+    const emailInput = screen.getByLabelText(/signIn.email.field/);
+    const passwordInput = screen.getByLabelText(/signIn.password.field/i);
+    const submitButton = screen.getByRole("button", { name: /signIn.button/i });
```

Comment:
also here. Why we should to use `/i` flag for translations ?

Reply:
Also, removed